### PR TITLE
Improve Node challenges wording and formatting

### PR DESCRIPTION
**Pull request by Atomk**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

Changed the descriptions of the Node challenges to achieve a bit higher level of quality. There were many inconstitencies in the way some technical terms are used, some confusing formatting errors due to words being put between angle brackets and ignored by the browser... A lot of stuff. I didn't fix everything, but now it is better for sure.

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi @Atomk thanks for the PR, can you please rebase this against staging? There are conflicts that have crept in with https://github.com/freeCodeCamp/freeCodeCamp/pull/14054

File: seed/challenges/05-apis-and-microservices/basic-node-and-express.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
...
       "description": [
-        "During the development process, is important to be able to check what’s going on in your code. Node is just a javascript environment. Like client side javascript, you can use the console to display useful debug information. On your local machine, you would see the console output in a terminal. On Gomix you can open the logs in the lower part of the screen. You can toggle the log panel if you push the button ‘Logs’ (top-left, under the app name).",
-        "To get started, just put the classic Hello World in the console. We recommend to keep the log panel open while working at these challenges. Reading the logs you can be aware of the nature of the errors that may occur."
+        "During the development process, it is important to be able to check what’s going on in your code. Node is just a Javascript environment. Like client side Javascript, you can use the console to display useful debug information. On your local machine, you would see the console output in a terminal. On Gomix you can open the logs in the lower part of the screen. You can toggle the log panel with the button ‘Logs’ (top-left, under the app name).",
```
Comment (raisedadead) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]:
```diff
- Javascript
+ JavaScript
```

As per convention.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: seed/challenges/05-apis-and-microservices/basic-node-and-express.json (hunk 1 above)
Comment (raisedadead) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]: Here and all other places.

**Comment by Atomk** [comment]
@raisedadead I did a bit of a mess with Git but I think I solved the problem.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @Atomk

**Pull request merged**